smac/initial_design/multi_config_initial_design.py

```diff
@@ -0,0 +1,113 @@
+import sys
+import typing
+import numpy as np
+
+from ConfigSpace.configuration_space import Configuration
+
+from smac.initial_design.initial_design import InitialDesign
+from smac.initial_design.single_config_initial_design import SingleConfigInitialDesign
+from smac.intensification.intensification import Intensifier
+
+from smac.tae.execute_ta_run import ExecuteTARun
+from smac.stats.stats import Stats
+from smac.utils.io.traj_logging import TrajLogger
+from smac.scenario.scenario import Scenario
+from smac.tae.execute_ta_run import StatusType
+from smac.runhistory.runhistory import RunHistory
+from smac.utils import constants
+
+
+
+__author__ = "Marius Lindauer"
+__copyright__ = "Copyright 2016, ML4AAD"
+__license__ = "3-clause BSD"
+
+
+class MultiConfigInitialDesign(InitialDesign):
+
+    def __init__(self,
+                 tae_runner: ExecuteTARun,
+                 scenario: Scenario,
+                 stats: Stats,
+                 traj_logger: TrajLogger,
+                 runhistory: RunHistory,
+                 rng: np.random.RandomState,
+                 get_configs: typing.Callable,
```

Comment:
1. This is motivated by the code in the single_config_intial_design, where we have the `_select_configuration` method to get the configuration. So, it is more consistent to do it a similar way.
2. If we want to later implement another initial design with an internal function using the ConfigSpace, we could simply implement a corresponding function.